Source/Core/UICommon/GameFile.cpp

```diff
@@ -222,9 +222,11 @@ bool GameFile::BannerChanged()
   if (!DiscIO::IsWii(m_platform))
     return false;
 
-  m_volume_banner.buffer =
-      DiscIO::WiiSaveBanner(m_title_id).GetBanner(&m_volume_banner.width, &m_volume_banner.height);
-  if (m_volume_banner.buffer.empty())
+  m_pending.volume_banner.buffer =
+      DiscIO::WiiSaveBanner(m_title_id)
+          .GetBanner(&m_pending.volume_banner.width, &m_pending.volume_banner.height);
+
+  if (m_pending.volume_banner.buffer.empty())
     return false;
 
   // We only reach here if m_volume_banner was empty, so we can always return true
```

Comment:
I missed this yesterday, but `m_volume_banner` should be `m_pending.volume_banner` now, I think

Alternatively, you could just return `m_pending.volume_banner.empty() != m_volume_banner.empty()`.

Reply:
No, the comment is correct. We only reach here if `m_volume_banner` is empty and `m_pending.volume_banner` is not empty. Maybe I should reword the comment, though...

Reply:
Eh, I'll just go with your latter suggestion.